packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -1447,6 +1449,7 @@ export default class TradeStore extends BaseStore {
             this.prev_chart_layout.is_used = false;
         }
         this.resetAccumulatorData();
+        this.setBarrierChoices([]);
```

Comment:
@kate-deriv we have 2 options here: 
1) add a check for is_vanillas in order to not reset selected barrier value on page reload for turbos,
2) add this change to your chacklist saying that selected barrier value will be reset on page reload both for turbos and vanillas.
please choose :)

Reply:
I'll stick to the 1st option in order to incapsulate changes, thank you for suggestion